Explorer/Assets/DCL/DemoWorlds/Extensions/NullExtensions.cs

```diff
@@ -0,0 +1,15 @@
+using System;
+
+namespace DCL.DemoWorlds.Extensions
+{
+    public static class NullExtensions
+    {
+        public static T EnsureNotNull<T>(this T? value, string? message = null)
```

Comment:
Does this extension need to be moved to a more genralised extension location? It isn't fully related to DemoWorlds? Does it need constraints for references only?

Reply:
Good suggestion, where is the general extension assembly?

Reply:
You may have to create it - I don't know!

Reply:
So I'll ask in chats to the team, maybe we already have one